### ISSUE TITLE
[Bug Report][3.7.11] Screen reader is not announcing the sorting information for the column headers under Nutrition section.

### ISSUE THREAD

**Issue by msftedad** [vuetifyjs/vuetify#20974]
### Environment
**Vuetify Version:** 3.7.11
**Vue Version:** 3.5.13
**Browsers:** Edge 133.0.0.0
**OS:** Windows 10

### Steps to reproduce

1. Open the URL in the latest edge browser.
2. Turn on the screen reader and press insert + space bar to change the screen reader mode/.
3. Press short cut key 't' to move to the table.
4. Use 'ctrl + alt + arrow keys' to move to the column header of the table and activate it using enter key.
5. Verify whether screen reader is announcing any information after activating the sorting control present in column headers of the table under Nutrition section or not.

### Expected Behavior
Screen reader should announce the sorting information after activating the sorting control present in column headers of the table under Nutrition section in Windows. 
Ensure that Screen reader is announcing as "calories button". on sorting the column header screen reader should be announcing the sorting information as "calories ascending/descending"

### Actual Behavior
Screen reader is not announcing any information after activating the sorting control present in column headers of the table under Nutrition section in Windows. It remains silent.
Observation: Screen reader is not announcing the role and state of the sortable column headers as well.

https://github.com/user-attachments/assets/c6f86c4e-3ff7-4240-b068-3df4d9f94d90

<img width="969" alt="Image" src="https://github.com/user-attachments/assets/2b488da8-b4c9-4701-9f16-078a9b60e595" />

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqtlltr2zAUgP+KMIy2ELu5tzFZYbS7dB1jsKfR9EGxTxKtsmRkOWsW8t93FNuJ7yQwP0nnpk/nHMl63lofwtBZx2C51lRDEHKq4W4mCJmubY8q3wwJ0UxzeD+zvsdaMc2kmFmJYoH2ZrR3QacsBFnbEZfa1fCmU90+pJnbCwY8DWy+tR1IHziGj4Aqb5XFNh+n873mZ0UTKghB+DYTApTNPCnQLPCZHdClYItN3nZNFaNCo4GMNWcC/Lx2xXywfdCU8egojZhYcrCNdSa8m17nd5Dt+TqXt1S0tn2qqa3pnB+83RVQH1SEFOnoCOEyjGE0PkQRKJ1XJUmpZGcPc1wlqRmKTNFwUqKaRp5iYVIKeAul0sSHBY25JtsknglFLq+yKSEKdKzEcY4p2QO45OKik8tesheXPB9lJO9lPsrZUqBjpKnSeW/zvcIGVYIGUNZEyGk255IF5RGUtPumRM+HJGfkstftLhFSrbF0V8VYu8Jsm67pUS4Vg+iicwh2n4kaXBYU+Q/Wnygui2s1xlfzYnCctzmESmpgIufyI5G0OTEl8x6POCWX70rmL7lx1mTtNTMFMVtU8i8I8ksu42rpsgS6pDealHSYKZeMnW7FBXPgkv6wJE+37pJhxcVsEFdoK2gt+qMHxFNAAxJR4f9h3qqZvz+4qeOfNPFXzHP8g//E/9HjlKkW5nG/jrnXnPQy2QG66pJAF3d5CvR9HHr0tXKUj9SD7qiOeuCUM5pCj8/O9O3Z0J/xxgA1x17xW8BH4/PSPSwfiQP5wCmr0h4prnAK+lfgfEPmQPEOaCS/qU15twl80ng4qy4JeFF6Cvc3yTkLZdhCPalt765TFmfUxbrnqWuZi2FOYf4iBWw8GcyboYfdMkXa3Q3Qt43dPXbKRUuwh0XxKdwPUsQtd/dwVJvo/qipP0bFu6xwJOsbu39+sp+YfjI/2ybqUa820/3G4zgu5/NAXa5Bwtx8Fo8/010ySJQ4mV6nDy2rY+HT8TVyfkdS4ON6O7O8CJ917vPLDnUsMG8wfKuGOYNEaIy2u52161gDZ+j0b9AcL0en17M6WsXw8g98BvAt)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
